src/thor/route_matcher.cc

```diff
@@ -191,7 +191,7 @@ bool expand_from_node(const mode_costing_t& mode_costing,
         elapsed += cost;
         // overwrite time with timestamps
         if (use_timestamps)
-          elapsed.secs = shape[index].time() - shape[0].time();
+          elapsed.secs = shape.Get(index).time() - shape.Get(0).time();
```

Comment:
hmmm, how can I test this so that codecov would be contented?

Reply:
that shouldnt be your burden. we can merge and just ignore codecov in this instance. this is for the fast exact map matching endpoint which as a whole could use more structured testing.